ts/components/IOMarkdown/renderRules.tsx

```diff
@@ -160,7 +159,7 @@ export const DEFAULT_RULES: IOMarkdownRenderRules = {
    */
   Strong(strong: TxtStrongNode, render: Renderer) {
     return (
-      <Text key={getTxtNodeKey(strong)} style={{ fontWeight: "800" }}>
```

Comment:
I've a doubt: Why you have change this fontWeight from 800 to 600? Because the "bold" size is no longer supported?

Reply:
@Ladirico The `Bold` font weight is represented by the `700` value, so by setting `800` we are setting an even bolder weight. With the new `Titillio` typeface, this new weight of `800` results in an `ExtraBold` style. To illustrate, it's even more visible than a headline style. So we'll render the `Strong` style with the `Semibold` style because it's more subtle.

Reply:
thanks!